user_guide_src/source/extending/core_classes/003.php

```diff
@@ -4,7 +4,7 @@
 
 use CodeIgniter\Router\RouteCollection;
 
-class RouteCollection extends RouteCollection
+class MyRouteCollection extends RouteCollection
```

Comment:
We don't need `My` because we use namespacing, but yes this class has syntax error.

Reply:
Why don't you add `use CodeIgniter\Router\RouteCollection as BaseRouteCollection;` as in the next sample code?
https://codeigniter4.github.io/CodeIgniter4/extending/core_classes.html#extending-core-classes